packages/field-plugin/helpers/vite/src/utils/text.ts

```diff
@@ -0,0 +1,12 @@
+const styles = {
+  reset: '\u001b[0m',
```

Comment:
There is also a library for having colorful outputs in your console logs. I believe it is this one (https://www.npmjs.com/package/chalk)  we are also using in the CLI

Reply:
I'll check it out @BibiSebi. I saw the CLI uses it but have never tried it on my own. 🤔

Reply:
Yeah tbh I'd prefer using the library instead of keeping the color codes here. And it's not increasing our bundle size either (only for development mode).

But if you add `chalk`, you should add it to the vite config's `externals` as well. (Let me know if you don't understand what that is for)